### PR TITLE
Tv Maze website

### PR DESCRIPTION
**Pull request by yosefHesham**
### Brief of the PR

We created a website that fetches data from movies API and displays them to the user.

---

**Changes Made:**

- Created page to display movies.
- Created a module to fetch data from the Movies API.
- Created module to send data to the involvement API.
- Created movie card, each hard has a like button and comments section.
- Created a popup for each movie to show more details and comments.
- Created a function that counts the number of likes for each movie.
- Created a function that counts the number of comments for each movie.
- Created tests to test counter functions

[Recorded video of the website](https://www.loom.com/share/88fe20ad359a4063b2da9eb6ad8c83d6)

### CODE REVIEW

File: scripts/create-env.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const fs = require('fs');
+
+fs.writeFileSync('./.env', `API_KEY=${process.env.API_KEY}\n`);
+fs.writeFileSync('./.env', `APP_ID=${process.env.APP_ID}\n`);
```
Comment (brytebee) [inline scripts/create-env.js:4]: - [ ] **Optional:** These keys should be ignored in a more realistic development setting. They can compromise the security of your app very quickly. 😠

File: modules/movies_ui.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    </div>
+  </article>`;
+
+  /*
+  1- Each html element has query selector function so you can get any element inside it
+  2- you cann pass functions as arguments.
+  3- Ex: we have like-button element inside that temp.
+  4- we can get that button from the temp becauze it its parent
+  */
+  // configureListener(tmp, handleSubmit, { childClassName: '.button1' });
```
Comment (brytebee) [inline modules/movies_ui.js:39]: - [ ] **Required:** Kindly remove all unused code from your codebase. It makes your work look more professional.

File: modules/movies_ui.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
+import configureListener from './configure_listeners.js';
+import { handleComment, handleLike } from './event_listeners.js';
+
+import { toJson } from './json_helper.js';
+import LikesService from './likes_service.js';
+// import Movie from './movie';
```
Comment (brytebee) [inline modules/movies_ui.js:6]: - [ ] Required: Kindly remove all unused code from your codebase. It makes your work look more professional.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @yosefHesham & @bizip👋, 

Good job so far!

### Highlight
- Good video presentation.✔️ 
- App is deployed and works correctly.✔️
- ES6 syntax is used nicely and consistently throughout the app.✔️
- All tests pass.✔️
- The app is broken into modules and well managed.✔️

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification. 
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: modules/movies_ui.js (hunk 3 above)
Comment (yosefHesham) [reply modules/movies_ui.js:6]: Thank you for your feedback, but those couldn't be optional? 🤔😂
@brytebee

Comment (yosefHesham) [reply modules/movies_ui.js:6]: I mean all the comments

File: modules/movies_ui.js (hunk 2 above)
Comment (yosefHesham) [reply modules/movies_ui.js:39]: I was explaining to my partner how this function works but yeah you are right ✨✌️

**Review by zhadier — APPROVED** [review]
# Hi @yosefHesham &  @bizip  👋 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



## Good luck and keep rocking! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**